tests/functional/aws-node-sdk/test/bucket/deleteBucketPolicy.js

```diff
@@ -18,17 +23,16 @@ const bucketPolicy = {
 };
 
 // Check for the expected error response code and status code.
-function assertError(err, expectedErr, cb) {
+function assertError(err, expectedErr) {
```

Comment:
Probably something we can dedup if we have it in otherfiles (like the one i commented above)